src/plugins/commit-message/index.js

```diff
@@ -30,25 +35,41 @@ const EXCLUDED_REPOSITORY_NAMES = new Set([
  */
 function checkCommitMessage(message) {
```

Comment:
If we are only returning errors, maybe this should be renamed to getCommitMessageErrors?